### ISSUE TITLE
Unauthorized, unwanted Locality modifications

### ISSUE THREAD

**Issue by acdoll** [ArctosDB/arctos#5139]
Something weird just happened when I edited a collecting event and locality.  see: https://arctos.database.museum/guid/DMNS:Mamm:9200
I went into Collecting Event  and corrected the verbatim locality to match what we actually have on our data card. I then went into the locality to change the county from Douglas to Arapahoe. When I did that the coordinates were automatically changed (by the georeferencing bot?). 
![image](https://user-images.githubusercontent.com/12507297/194428511-4e96a9fc-088e-46ff-9683-5b82b34647e3.png)

Why?? The coordinates that were in there were good. It just decided to input coordinates for Littleton, CO without my knowledge. These coordinates and their uncertainty don't even include the specific location we had already determined. 
This is not good.  What's going on? Fortunately I check my work as I go - not everyone does this. 

BTW: the polygons for these counties are not accurate - they miss all the fine detail along the boarders.

**Comment by dustymc** [comment]
Duplicate of https://github.com/ArctosDB/arctos/issues/5137 - I'm working on a rollback, NOT fixing things would make my life a bit easier.

**Comment by acdoll** [comment]
Thanks Dusty! Sorry if my message sounded a little snippy - this was just one of many frustrations for my day.

**Comment by dustymc** [comment]
No problem, nothing you said was at all wrong.

The bot will do crazy things from time to time, but it should never replace information, just add - and then the crazy things will make it easier to find problems/focus your attention.

I can try replacing the spatial with GADM, but it's always (except maybe in the really square stuff - and usually even then, check out the southern border of NE sometime) going to be imperfect - https://en.wikipedia.org/wiki/Coastline_paradox (and a bit of lost surveyors).

If I was a bit closer we'd have to find something to dump over a single cube of ice after this afternoon....

**Comment by sharpphyl** [comment]
@dustymc 

I understand the value of the georeference_bot, but I don't always want the locality to be geolocated.  For the record https://arctos.database.museum/guid/DMNS:Inv:8702, we have a locality but low confidence in our interpretation of what it means.  Therefore, I did not want to geolocate the specimen even though I entered the data with a remark describing the  insufficiency of the data.   

Is there a way to choose whether we want the bot coordinates based on the locality text?
